docs/rules/camelcase.md

```diff
@@ -70,6 +73,16 @@ var obj = {
 };
 ```
 
+### ignorePattern
+
+Examples of correct code for this rule with the `{ "ignorePattern": true }` option:
+
+```js
+/*eslint camelcase: ["error", {"ignoreStrings": ["ignored_.*"]}]*/
```

Comment:
I'm confused by the documentation for this schema. The general option description implies that the schema is `ignorePattern: string[]`. The text above this example implies that the schema is `ignorePattern: boolean`. The inline comment in this example implies that the schema is `ignoreStrings: string[]`. The rule implementation sets the schema to `ignorePattern: string`.